sql/core/src/test/resources/sql-tests/results/pgSQL/float4.sql.out

```diff
@@ -63,31 +63,31 @@ SELECT float('nan')
 -- !query 7 schema
 struct<CAST(nan AS FLOAT):float>
 -- !query 7 output
-NULL
+NaN
```

Comment:
Please remove `SPARK-28060` in `float4.sql`. That is a duplicate of SPARK-27768 (this PR).